src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -41,3 +65,62 @@ export class SurgingTempest extends Analyser {
 		return (statusUptime / fightUptime) * 100
 	}
 }
+
+export class StormsEye extends CoreGauge {
+	static override handle = 'stormseye'
+
+	@dependency private suggestions!: Suggestions
+
+	private surgingTempest = this.add(new TimerGauge({
+		maximum: 60,
+	}))
+
+	private earlyRefreshCount = 0
+
+	override initialise() {
+		super.initialise()
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.action(this.data.matchActionId(SURGING_TEMPEST_GENERATORS)),
+			(_) => this.extendSurgingTempest(SURGING_TEMPEST_GENERATION_AMOUNT)
+		)
+
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.action(this.data.matchActionId(SURGING_TEMPEST_EXTENDERS)),
+			(_) => this.extendSurgingTempest(SURGING_TEMPEST_EXTENSION_AMOUNT, true)
+		)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private extendSurgingTempest(amount: number, onlyIfRunning ?: boolean) {
```

Comment:
You can just make the boolean field default to false, or alternatively remove it entirely and have a local var for it - if it's IR, we set it to true, otherwise false.